modules/fbx/data/fbx_skeleton.cpp

```diff
@@ -69,7 +69,7 @@ void FBXSkeleton::init_skeleton(const ImportState &state) {
 			// Make sure the bone name is unique.
 			const String bone_name = bone->bone_name;
 			int same_name_count = 0;
-			for (int y = x; y < skeleton_bone_count; y++) {
+			for (int y = x + 1; y < skeleton_bone_count; y++) {
```

Comment:
This fixes #43820 right?

Reply:
yep

Reply:
yeah we already merged that in 3 I think

Reply:
I extracted it in its own PR #47458 as this one is still WIP and will likely not make it in time for 3.3-stable.

Reply:
Ah you were right @BastiaanOlij, this has already been fixed in `3.x` via 9c296bcb59c. Should really have gone to `master` and be cherry-picked instead :)

Reply:
yeah when I did that one Gordon said he was already including it in one of his fixes so :)